content/posts/5-rad-dla-juniora.mdx

```diff
@@ -0,0 +1,39 @@
+- title: '5 rad dla juniora – jak podejść do pierwszej pracy'.
+  Category: 'Praca'
+  published at: '20-12-2021'
+  is published: false
+  popular: false
+  image: '/images/antywzorce-CSS/thumbnail. PNG'.
+  Excerpt: 'Początek przygody jako programista rzadko kiedy jest bezstresowy. Nagle trafia się do projektu bez jakiegokolwiek doświadczenia…'
+-
+
+Początek przygody jako programista rzadko kiedy jest bezstresowy.
+Nagle trafia się do projektu bez jakiegokolwiek doświadczenia, trzeba zdobyć teraz wiedzę domenową, dowiedzieć się kto jest kim, itp.
+Dlatego by właśnie zminimalizować możliwe źródła stresu, dam ci parę rad na dobry początek.
+No to lecimy!
+
+## Proszenie się o pomoc to nic złego!
+
+Niejednokrotnie na początku kariery będziesz napotykać taski oznaczone jako te „najłatwiejsze”, które przyblokują cię na dłuższy moment, bo w końcu nie posiadasz jeszcze odpowiedniej wiedzy domenowej bądź technicznej i jest to całkowicie normalne. Jednak nie popełniaj dość częstego błędu, którym jest pewnego rodzaju strach przed proszeniem o pomoc. Nie myśl, że twój zespół miałby cię z jakiegoś powodu wyśmiać, że nie umiesz zrobić X lub Y, bądź w ogóle zwolnić (tak, znam osoby co tak niestety, myślały). Każdy wie, że zostałeś zatrudniony jako junior i dobry zespół z wielką ochotą siądzie z tobą do taska, nakieruje na jego możliwe rozwiązanie i wytłumaczy parę kwestii, które cię dręczą. Kto pyta, nie błądzi!
```

Comment:
Prawdopodobnie na początku kariery będziesz wykonywał taski oznaczone jako te „najłatwiejsze”, ale nie dołuj się, jeśli przyblokują Cię one na dłuższy moment. To całkowicie normalne, w końcu nie posiadasz jeszcze odpowiedniej wiedzy domenowej/technicznej Twojego nowego projektu, na to potrzeba czasu. Jeśli napotkasz na jakiś problem w żadnym wypadku nie bój się poprosić kogoś o pomoc. Każdy wie, że jesteś nową osobą w projekcie, ktoś na pewno nakieruje Cię na jego rozwiązanie problemu i wytłumaczy parę kwestii, które cię dręczą, pamiętaj, kto pyta, nie błądzi!